test_mpop_service/TestMpopService.cpp

```diff
@@ -44,7 +44,7 @@ void TestMpopService::removeDatabaseTestEntries() {
 void TestMpopService::cleanupTestCase()
 {
     if (this->is_mysql_supported) {
-        this->facade->deleteAllFromDatabase();
+        //this->facade->deleteAllFromDatabase();
```

Comment:
We should call `deleteTagsVisitorsAndTheirAnswers(const QList<QString>& rfidTags)` here.